app/src/main/res/values-de/strings.xml

```diff
@@ -644,6 +644,7 @@ Signatur: %5$s</string>
     <string name="chat_message_try_again">Die Nachricht konnte nicht gesendet werden. Erneut versuchen?</string>
     <string name="leave">Verlassen</string>
     <string name="today">Heute</string>
+    <string name="ticket_sale_beta">Der Ticketverkauf ist still in der Beta Phase und es können Fehler auftreten. Solltet ihr Probleme haben, könnt ihr uns unter app@tum.de erreichen.</string>
```

Comment:
"Der Ticketverkauf ist still in der Beta Phase" -> kein richtiges Deutsch
ich würde vorschlagen: "Der ticketverkauf befindet sich momentan im Testbetrieb"